src/app/(auth-required)/components/notice/Modal.tsx

```
@@ -0,0 +1,46 @@
+
+      {data?.posts?.map(({ content, created_at, id, title }) => (
+        <div key={id} className="flex flex-col gap-2">
+          <div className="flex items-center gap-3">
+            <h3 className="text-TEXT-MAIN text-T4 max-MBI:text-T5">{title}</h3>
+            <h4 className="text-TEXT-ALT text-T5 max-MBI:text-ST5">
+              {created_at.split('T')[0]}
+            </h4>
+          </div>
+          <div
+            className="text-TEXT-MAIN text-I4"
+            dangerouslySetInnerHTML={{ __html: content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)